src/node_jsvmapi.cc

```diff
@@ -204,7 +204,7 @@ namespace v8impl {
     public:
       CallbackWrapperBase(const T& cbinfo)
         : _cbinfo(cbinfo),
-          _cbdata(cbinfo.Data().As<v8::Object>()) {
+          _cbdata(v8::Local<v8::Object>::Cast(cbinfo.Data())) {
```

Comment:
Why is a cast necessary? Only our own internal code can invoke this constructor, and it always creates an Object to hold the callback data.